cantabular-import/README.md

```diff
@@ -240,4 +240,26 @@ On the Docker network we have a Kowl as our UI to monitor our Kafka Cluster on t
 The tools display information such as brokers, topics, partitions, consumers, message schemas and 
 lets you view messages on our docker network.
 
-* [**Kowl**](https://github.com/redpanda-data/kowl) - [http://localhost:9888](http://localhost:9090)
\ No newline at end of file
+* [**Kowl**](https://github.com/redpanda-data/kowl) - [http://localhost:9888](http://localhost:9090)
+
+# Developing services and libraries with the stack
+
+While developing service code any changes will be automatically picked up by `reflex` when files are
+saved and the services will be automatically rebuilt.
+
+When developing a library that is used by the services as a dependency (i.e. `dp-api-clients-go`,
+`dp-net` etc) a couple of extra steps are needed to pick up local changes. Firstly, in order to
+point the service to use your local copy of the library you will need to add a replace directive
+in `go.mod` pointing to the local copy you're working on. For example
+
+`replace github.com/ONSdigital/dp-api-clients-go/v2 => /Users/FlorenceRoundabout/path/to/dp-api-clients-go`
```

Comment:
You can also use relative paths, assuming that the lib is on the same workspace has your other project/repo/microservice
`replace github.com/ONSdigital/dp-api-clients-go/v2 => ../dp-api-clients-go`